<launch>
<arg name="connect_str" default="tcp:$(env SITL_IP):5760" />
<arg name="sysid" default="1" />
<node pkg="macaw" name="macaw" exec="macaw" >
  <param name="mavlink_sysid" value="$(var sysid)" />
  <param name="mavlink_connect_str" value="$(var connect_str)" />
</node>
</launch>